pack/pack.go

```diff
@@ -0,0 +1,61 @@
+package pack
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"k8s.io/helm/pkg/chartutil"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	PackageHelmChartTaskName = "package-helm-chart"
+
+	PackageHelmChartTaskString = "%s: %s"
+)
+
+type PackageHelmChartTask struct {
+	chartDir string
+	dst      string
+}
+
+// Run package the helm chart at p.chartDir into p.dst.
+//
+// If p.dst is /foo, and the chart is named bar, with version 1.0.0, this
+// will generate /foo/bar-1.0.0.tgz.
+func (p PackageHelmChartTask) Run() error {
+	path, err := filepath.Abs(p.chartDir)
+	if err != nil {
+		return err
+	}
+
+	// Load chart from a directory.
+	ch, err := chartutil.LoadDir(path)
+	if err != nil {
+		return err
+	}
+
+	// Save the chart as an archive in the given director.
```

Comment:
I would just skip those comments.

```suggestion
	// Save the chart as an archive in the given directory.
```